security-analysis/security-analysis-api/src/main/java/com/powsybl/security/LimitViolationHelper.java

```diff
@@ -24,7 +24,10 @@ private static VoltageLevel getVoltageLevel(LimitViolation limitViolation, Netwo
         Objects.requireNonNull(limitViolation);
 
         Identifiable identifiable = network.getIdentifiable(limitViolation.getSubjectId());
-        if (identifiable instanceof Branch) {
+        if (identifiable instanceof Switch) {
```

Comment:
No switch, see previous comment.

Reply:
Done. Switches are now not allowed as TerminalRef.